### ISSUE TITLE
Store whitelist in the database and add management API

### ISSUE THREAD

**Issue by aleksey-novikov** [crowdsecurity/crowdsec#1650]
### What would you like to be added?

Add support to store whitelists in the database and query them in the postoverflows by expr. Also add API for managing whitelists.

### Why is this needed?

Currently there is no method to permanently unblock a blocked address. In some cases it is required to make an exception from scenarios for certain clients

**Comment by LaurenceJJones** [comment]
Hey, it depends on which address is currently being blocked, if it is just a LAPI decision you can use a whitelist to prevent your local LAPI from blocking the address: https://docs.crowdsec.net/docs/next/whitelist/intro. If the address is coming down from the CAPI these don't take the whitelist into consideration. So I believe this to be a duplicate of #1239

**Comment by aleksey-novikov** [comment]
duplicate of #1239
